### PR TITLE
Istio lab4 and lab5 changes

### PR DESCRIPTION
**Pull request by lee-zhg**
1) Istio lab4 README.md.
2) Istio lab5 README.md 
3) plans/mixer-rule-denial.yaml

### CODE REVIEW

File: enforce-policies/README.md [hunk 1]
```diff
@@ -52,14 +52,16 @@ In this exercise we'll use the denier adapter.
 
 2. Verify that the service is denied:
 
-   In [this exercise](../create-ingress-gateway/README.md), we created the Ingress Gateway resource. Make sure the $INGRESS_IP environment variable is still present. Then in the terminal, try:
+   In [Exercise 5](../exercise-5/README.md), we created the Ingress resource. Make sure the $INGRESS_IP environment variable is still present. Then in the terminal, try:
```
Comment (timroster) [inline enforce-policies/README.md]: the repo targeted by this PR does not have a `../exercise-5/README.md` file nor does the PR create one so this needs to be amended to the original.

**Review by timroster — CHANGES_REQUESTED** [review]
@lee-zhg one of the changes to `enforce-policies/README.md` has a problem I think. Please check.

File: enforce-policies/README.md (hunk 1 above)
Comment (timroster) [reply enforce-policies/README.md]: LGTM

**Review by timroster — APPROVED** [review]

File: enforce-policies/README.md (hunk 1 above)
Comment (lee-zhg) [reply enforce-policies/README.md]: @timroster README.md file was updated. I also replace the .yaml file in the README.md file

**Pull request merged**